playbooks/common/openshift-cluster/upgrades/upgrade_control_plane.yml

```diff
@@ -173,7 +173,9 @@
   - name: Reconcile Cluster Roles
     command: >
       {{ openshift.common.client_binary }} adm --config={{ openshift.common.config_base }}/master/admin.kubeconfig
-      policy reconcile-cluster-roles --additive-only=true --confirm
+      policy reconcile-cluster-roles --additive-only=true --confirm -o name
+    register: reconcile_cluster_role_result
+    changed_when: reconcile_cluster_role_result.stdout.length > 0
```

Comment:
Missing part of the acceptance criteria

* The `changed_when` parameter is added to each task such that it is only `true` when there is **output from the task** and the **return code is `0`**

For example:

```yaml
   - name: Reconcile Cluster Roles
      command: >
        {{ openshift.common.client_binary }} adm --config={{ openshift.common.config_base }}/master/admin.kubeconfig
      policy reconcile-cluster-roles --additive-only=true --confirm
      policy reconcile-cluster-roles --additive-only=true --confirm -o name
    register: reconcile_cluster_role_result
    changed_when: 
    - reconcile_cluster_role_result.stdout.length > 0
    - reconcile_cluster_role_result.rc == 0
```

would be more like what we're looking for